src/components/Modal/index.tsx

```diff
@@ -215,16 +213,16 @@ const StyledDialog = styled(Dialog, {
       props.toString(),
     ),
 })<StyledDialogProps>`
-  background-color: ${({ theme }) => theme.colors.neutral.backgroundWeak};
+  background-color: ${({ theme }) =>
+    theme.colors.neutral.backgroundWeakElevated};
   position: relative;
   border-radius: ${({ bordered }) => (bordered ? 4 : 0)}px;
   border: 0;
   padding: 32px;
   ${({ placement }) => MODAL_PLACEMENT[placement]}
   width: ${({ width }) => MODAL_WIDTH[width]}px;
   min-height: ${({ height }) => height};
-  box-shadow: 0 0 12px 18px
-    ${({ theme }) => transparentize(0.8, theme.colors.neutral.backgroundStrong)};
+  box-shadow: 0 0 12px 18px #151a2d5c;
```

Comment:
From what I see in shadows tokens it should be more like:

```suggestion
  box-shadow: 0 8px 40px 10px #151A2D5C;
```

It should make a more beautiful shadow in light theme